kernelci/build.py

```diff
@@ -286,6 +287,20 @@ def make_tarball(kdir, tarball_name):
     subprocess.check_output(cmd, shell=True)
 
 
+def extract_tarball(dest_path, tarball_name):
```

Comment:
Why not just do that as part of `pull_tarball()`?

Reply:
removed and code included in pull_tarbal()